test/validate/ferc1_test.py

```diff
@@ -98,14 +101,30 @@ def test_minmax_rows(pudl_out_ferc1, live_dbs, expected_rows, df_name):
 @pytest.mark.parametrize(
     "df_name,unique_subset", [
         ("pu_ferc1", ["utility_id_ferc1", "plant_name_ferc1"]),
-        ("fbp_ferc1", ["report_year", "utility_id_ferc1", "plant_name_ferc1"]),
-        ("plants_hydro_ferc1",
-         ["report_year", "utility_id_ferc1", "plant_name_ferc1", "capacity_mw"]),
-        ("plants_pumped_storage_ferc1",
-         ["report_year", "utility_id_ferc1", "plant_name_ferc1", "capacity_mw"]),
-        ("plant_in_service_ferc1",
-         ["report_year", "utility_id_ferc1", "amount_type"]),
-    ])
+        ("fbp_ferc1", [
+            "report_year",
+            "utility_id_ferc1",
+            "plant_name_ferc1"
+        ]),
+        ("plants_hydro_ferc1", [
+            "report_year",
+            "utility_id_ferc1",
+            "plant_name_ferc1",
+            "capacity_mw"  # Why does having capacity here make sense???
```

Comment:
I assume there are duplicates based on the `"report_year", "utility_id_ferc1", "plant_name_ferc1"` and the cpacity distinguishes them... this makes me wonder why we are compiling all of these lists of columns that are in effect the PKs of the tables. Why wouldn't we just pull these directly from the metadata. And adjust when necessary bc obviously these are not all analogous to db tables.

The intention of this comment is to maybe lead to an issue for a future enhancement - not to lead to changes in this PR/release.

Reply:
Previously we didn't have many of the primary keys actually encoded. Yes, now that we've got them available in the metadata, I think this can be done automatically (for the EIA stuff, not for FERC 1 which is still keyless).

In these comments I'm questioning whether checking for uniqueness even makes sense -- these tables don't have natural primary keys, and I don't know what exactly the argument is for using capacity to fake one.